openstack/sharedfilesystems/v2/sharetransfers/results.go

```diff
@@ -0,0 +1,167 @@
+package sharetransfers
+
+import (
+	"encoding/json"
+	"net/url"
+	"strconv"
+	"time"
+
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+const (
+	invalidMarker = "-1"
+)
+
+// Transfer represents a Share Transfer record.
+type Transfer struct {
+	ID                   string              `json:"id"`
+	Accepted             bool                `json:"accepted"`
+	AuthKey              string              `json:"auth_key"`
+	Name                 string              `json:"name"`
+	SourceProjectID      string              `json:"source_project_id"`
+	DestinationProjectID string              `json:"destination_project_id"`
+	ResourceID           string              `json:"resource_id"`
+	ResourceType         string              `json:"resource_type"`
+	CreatedAt            time.Time           `json:"-"`
+	Links                []map[string]string `json:"links"`
+}
+
+// UnmarshalJSON is our unmarshalling helper.
+func (r *Transfer) UnmarshalJSON(b []byte) error {
+	type tmp Transfer
+	var s struct {
+		tmp
+		CreatedAt gophercloud.JSONRFC3339MilliNoZ `json:"created_at"`
```

Comment:
Which makes me think, we still have https://github.com/gophercloud/gophercloud/issues/2507 :sweat_smile: